concent_api/concent_api/celery.py

```diff
@@ -0,0 +1,17 @@
+import os
+from celery import Celery
+
+# set the default Django settings module for the 'celery' program.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'concent_api.settings')
+
+
+app = Celery('concent_api')
+
+# Using a string here means the worker don't have to serialize
```

Comment:
`Using a string here means that the worker does not have to serialize`

Reply:
Done